packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -164,7 +164,7 @@ export const DeviceDisplay = ({ address, valueDataTest }: DeviceDisplayProps) =>
             const PAGE_SIZE = 4;
             return (
                 <>
-                    <Chunks onCopy={handleOnCopy} data-test={valueDataTest}>
+                    <Chunks onCopy={handleOnCopy}>
```

Comment:
I understand why you add data-test to not-paginated chunks, but why you removed it from here?

Reply:
It was not the correct place where it should be so I removed it from there and put it elsewhere where it suppose to be